\hypertarget{exceptions_8cpp}{}\doxysection{exceptions.\+cpp File Reference}
\label{exceptions_8cpp}\index{exceptions.cpp@{exceptions.cpp}}


Función principal del programa.  


{\ttfamily \#include $<$iostream$>$}\newline
{\ttfamily \#include \char`\"{}exceptions.\+hpp\char`\"{}}\newline
{\ttfamily \#include $<$stdexcept$>$}\newline
\doxysubsection*{Functions}
\begin{DoxyCompactItemize}
\item 
void \mbox{\hyperlink{exceptions_8cpp_afc7d39c09d322af3f8cd616424ed0304}{funcion\+Excepcion\+Logica}} ()
\begin{DoxyCompactList}\small\item\em Función principal del programa. \end{DoxyCompactList}\item 
\mbox{\Hypertarget{exceptions_8cpp_a4da194ae664d2b906407690d676b0a07}\label{exceptions_8cpp_a4da194ae664d2b906407690d676b0a07}} 
void {\bfseries funcion\+Excepcion\+Tiempo\+Ejecucion} ()
\begin{DoxyCompactList}\small\item\em Función que lanza una excepción de tipo std\+::runtime\+\_\+error. \end{DoxyCompactList}\item 
\mbox{\Hypertarget{exceptions_8cpp_a5763874ba73103c8574f92977ba95d97}\label{exceptions_8cpp_a5763874ba73103c8574f92977ba95d97}} 
void {\bfseries funcion\+Excepcion\+Argumento\+Invalido} ()
\begin{DoxyCompactList}\small\item\em Función que lanza una excepción de tipo std\+::invalid\+\_\+argument. \end{DoxyCompactList}\item 
\mbox{\Hypertarget{exceptions_8cpp_ae66f6b31b5ad750f1fe042a706a4e3d4}\label{exceptions_8cpp_ae66f6b31b5ad750f1fe042a706a4e3d4}} 
int {\bfseries main} ()
\begin{DoxyCompactList}\small\item\em Función principal del programa. \end{DoxyCompactList}\end{DoxyCompactItemize}


\doxysubsection{Detailed Description}
Función principal del programa. 

Este archivo contiene la función principal del programa, que lanza una excepción personalizada de la clase \mbox{\hyperlink{class_my_exception}{My\+Exception}} y la captura para imprimir información sobre ella. 

\doxysubsection{Function Documentation}
\mbox{\Hypertarget{exceptions_8cpp_afc7d39c09d322af3f8cd616424ed0304}\label{exceptions_8cpp_afc7d39c09d322af3f8cd616424ed0304}} 
\index{exceptions.cpp@{exceptions.cpp}!funcionExcepcionLogica@{funcionExcepcionLogica}}
\index{funcionExcepcionLogica@{funcionExcepcionLogica}!exceptions.cpp@{exceptions.cpp}}
\doxysubsubsection{\texorpdfstring{funcionExcepcionLogica()}{funcionExcepcionLogica()}}
{\footnotesize\ttfamily void funcion\+Excepcion\+Logica (\begin{DoxyParamCaption}{ }\end{DoxyParamCaption})}



Función principal del programa. 

Esta función es la primera en ejecutarse cuando se inicia el programa. Lanza una excepción personalizada de la clase \mbox{\hyperlink{class_my_exception}{My\+Exception}} y la captura para imprimir información sobre ella.

\begin{DoxyReturn}{Returns}
int Código de salida del programa
\end{DoxyReturn}
Función que lanza una excepción de tipo std\+::logic\+\_\+error 